tools/pipelines/templates/build-npm-package.yml

```diff
@@ -302,6 +302,8 @@ stages:
                   npx lerna exec --no-private --no-sort -- npm pack --unsafe-perm && \
                   npx lerna exec --no-private --no-sort --parallel -- mv -t $(Build.ArtifactStagingDirectory)/pack/scoped/ ./*.tgz && \
                   npx lerna exec --no-private --no-sort --parallel -- "[ ! -f ./*test-files.tar ] || (echo 'test files found' && mv -t $(Build.ArtifactStagingDirectory)/test-files/ ./*test-files.tar)"
+
+                  npx lerna ls --toposort | sed 's/@//' | sed 's/\//-/' | head -c -1 > $(Build.ArtifactStagingDirectory)/pack/packagePublishOrder.txt
```

Comment:
Nit: a comment explaining what this is doing might help.

Reply:
Done